core/services/ardupilot_manager/main.py

```diff
@@ -23,29 +28,44 @@ def render(self, content: Any) -> bytes:
         ).encode(self.charset)
 
 
-app = FastAPI(default_response_class=PrettyJSONResponse)
+app = FastAPI(
+    title="ArduPilot Manager API",
+    description="ArduPilot Manager is responsible for managing ArduPilot devices connected to Companion.",
+    default_response_class=PrettyJSONResponse,
+)
 autopilot = ArduPilotManager()
 
 
 @app.get("/endpoints", response_model=List[Dict[str, Any]])
+@version(1, 0)
 def get_available_endpoints() -> Any:
     return [endpoint.asdict() for endpoint in autopilot.get_endpoints()]
 
 
 @app.post("/endpoints", status_code=status.HTTP_201_CREATED)
+@version(1, 0)
 def create_endpoints(response: Response, endpoints: Set[Endpoint] = Body(...)) -> Any:
     if not autopilot.add_new_endpoints(endpoints):
         response.status_code = status.HTTP_409_CONFLICT
         return {"message": "One or more endpoints already exists or is not supported. No changes were made."}
 
 
 @app.delete("/endpoints", status_code=status.HTTP_204_NO_CONTENT)
+@version(1, 0)
 def remove_endpoints(response: Response, endpoints: Set[Endpoint] = Body(...)) -> Any:
     if not autopilot.remove_endpoints(endpoints):
         response.status_code = status.HTTP_409_CONFLICT
         return {"message": "One or more endpoints does not exist or is protected. No changes were made."}
 
 
+app = VersionedFastAPI(
+    app,
+    version="1.0.0",
+    prefix_format="/v{major}.{minor}",
```

Comment:
Use the 0.9.0 version of VersionedFastAPI and use `enable_latest`

Reply:
Done